### PR TITLE
Bump k8s modules to 0.25.0 and prepare for 2.7.0 release

### PR DESCRIPTION
**Pull request by shu-mutou**
Also, bump golang 1.19.

_This is in preparation for the next k8s release._
_K8s 1.25 would use golang 1.19_ https://github.com/kubernetes/kubernetes/pull/111254
_Need to wait for new release of k8s and golang._

[K8s v1.25 Release schedule](https://github.com/kubernetes/sig-release/blob/master/releases/release-1.25/README.md)

| status | version | date | remarks |
|---|-----|-----|-----|
| ✔ | 1.25.0-alpha.3 | Tuesday 19th July 2022 | np |
| ❌ | 1.25.0-beta.0 | Thursday 4th August 2022 | inconsistent dependencies |
| ✔ | 1.25.0-rc.0 | Tuesday 9th August 2022 | np |
| ✔ | 1.25.0-rc.1 | Tuesday 16th August 2022 | np |
| ✔ | 1.25.0 | Tuesday 23th August 2022 | np |

[Golang Releases](https://github.com/golang/go/tags) v1.19 released at Aug 3rd, 2022
[Go 1.19 Release Notes](https://tip.golang.org/doc/go1.19)

### CODE REVIEW

**Comment by shu-mutou** [comment]
Version ups for golang, golanci-lint and docker cli were divided into another PR.
So rebased and force pushed.

**Comment by shu-mutou** [comment]
@maciaszczykm PTAL

**Comment by maciaszczykm** [comment]
Checking it.

**Comment by maciaszczykm** [comment]
Works on 1.24, I still have to deploy 1.25 to check it.

File: i18n/messages.xlf [hunk 1]
```diff
@@ -1635,18 +1635,30 @@
           <context context-type="sourcefile">src/app/frontend/common/components/resourcelist/daemonset/template.html</context>
```
Comment (floreks) [inline i18n/messages.xlf:1635]: This could be reverted as there are no changes that impact i18n.

**Comment by maciaszczykm** [comment]
I have removed translation changes and bumped version to 2.7.0 as it will be good to release this once it will be tested.

**Comment by maciaszczykm** [comment]
Let's wait for wider adoption of this version so we will be able to test it properly.

**Comment by shu-mutou** [comment]
@maciaszczykm Please be patient while we call out to each translation team to update their translations.

**Comment by shu-mutou** [comment]
@headcr4sh @kgroschoff @mkorbi @raelga @anyulled @gochist @seokho-son @feloy @chenrui333 @zehuaiWANG @helight @hwdef @tanjunchen 
Hi translation teams!
Please response this message if you have will to update your translations for dashboard v2.7.0.
Thanks!

**Comment by shu-mutou** [comment]
@maciaszczykm 
Japanese translation team will create PR for translation update soon! Please wait for a few days.

**Comment by hwdef** [comment]
> @headcr4sh @kgroschoff @mkorbi @raelga @anyulled @gochist @seokho-son @feloy @chenrui333 @zehuaiWANG @helight @hwdef @tanjunchen Hi translation teams! Please response this message if you have will to update your translations for dashboard v2.7.0. Thanks!

Chinese translation team will also create PR for translation update. 
I will update the simplified Chinese translation.

**Comment by shu-mutou** [comment]
/hold
need to wait for translation updates

**Comment by anyulled** [comment]
> @headcr4sh @kgroschoff @mkorbi @raelga @anyulled @gochist @seokho-son @feloy @chenrui333 @zehuaiWANG @helight @hwdef @tanjunchen 
> Hi translation teams!
> Please response this message if you have will to update your translations for dashboard v2.7.0.
> Thanks!

Thanks for the mention, the Spanish team will also prepare a PR with updated translation

**Comment by seokho-son** [comment]
Thanks for this notification.
Korean contributors will also create PR for translation update. :)

**Comment by shu-mutou** [comment]
Japanese team updates Japanese translation on following PR. Please wait for a few days.
https://github.com/kubernetes/dashboard/pull/7415

**Comment by feloy** [comment]
> > @headcr4sh @kgroschoff @mkorbi @raelga @anyulled @gochist @seokho-son @feloy @chenrui333 @zehuaiWANG @helight @hwdef @tanjunchen
> > Hi translation teams!
> > Please response this message if you have will to update your translations for dashboard v2.7.0.
> > Thanks!
> 

I will work on the french translation. Thanks for the mention

**Comment by hwdef** [comment]
Simplified Chinese translation:
https://github.com/kubernetes/dashboard/pull/7416

**Comment by feloy** [comment]
> > > @headcr4sh @kgroschoff @mkorbi @raelga @anyulled @gochist @seokho-son @feloy @chenrui333 @zehuaiWANG @helight @hwdef @tanjunchen
> > > Hi translation teams!
> > > Please response this message if you have will to update your translations for dashboard v2.7.0.
> > > Thanks!
> 
> I will work on the french translation. Thanks for the mention

I'm not able to install a fresh new dev environment. 

When cloning the master branch and running 'npm ci', the installation of node-sass (v4.13.0 as specified in package.json) fails.

I can run 
npm install node-sass@6 (it seems node-sass v6 is compatible with node v16)
npm ci

But then, running `npm run fix:i18n` due to the wrong node-sass.

Any advice?

https://github.com/kubernetes/dashboard/issues/7418

**Comment by seokho-son** [comment]
Hi, I am a reviewer for Korean localization :)
Korean contributors are working on localizations based on the Issue #7424
All tasks will be finished soon. I will leave a comment when we are ready. 
(Thanks @shu-mutou for enabling workflows of PRs from the new Korean contributors)

**Comment by maciaszczykm** [comment]
Hi! Can you report statuses of translation updates? I think we can release soon.

**Comment by seokho-son** [comment]
Hi @maciaszczykm @shu-mutou 
Thank you for waiting for our tasks!! :)

Korean translation work for 2.7.0 release has been completed. (I just approved the last PR)
https://github.com/kubernetes/dashboard/issues/7424#issuecomment-1238113138

We have a plan to check the whole translated Korean UIs again, 
but regardless of our plan I think you can release 2.7.0.!

I'd like to shoutout to 
@bconfiden2 @dewble @rollony @Seo-yul @gy-ulbak96 @jinnypark9393 @BrendenHJH @onestone9900 @NayeonKeum @yuzin9712 @having-dlrow who contributed Korean localization in a collaborative way for the coming Dashboard 2.7.0 release.

**Comment by shu-mutou** [comment]
Hi @maciaszczykm !
K8s v1.25.1 released today, but modules have not be released on [go module site](https://pkg.go.dev/k8s.io/client-go?tab=versions) for now. I'd like to update to it, so please wait for a few days.

**Comment by maciaszczykm** [comment]
@shu-mutou I'd like to wait with that. We can do the next release after some time. PR that splits containers is almost ready and after we will release we can merge it and lift the code freeze.

**Comment by maciaszczykm** [comment]
/lgtm
/approve

I did final checkups on kind 1.25 cluster.

**Pull request merged**